packages/x-components/src/wiring/wires.factory.ts

```diff
@@ -129,6 +136,40 @@ export function wireDispatchWithoutPayload(action: string): AnyWire {
   return (observable, store) => observable.subscribe(() => store.dispatch(action));
 }
 
+/**
+ * Creates a wires factory that can create wires that will invoke the service methods.
+ *
+ * @param service - The service to invoke its methods.
+ * @returns A factory to create wires that invoke the service methods.
+ * @public
+ */
+export function wireService<SomeService>(service: SomeService): WireService<SomeService> {
+  return (method, payload?) => {
+    return observable =>
```

Comment:
The IDE is complaining about this `payload` as it thinks its type is unknown:

![image](https://user-images.githubusercontent.com/3502075/129314797-2f65849c-354e-460a-a430-7430155dfd3e.png)

Reply:
Change IDE:

![image](https://user-images.githubusercontent.com/68222542/129846241-72710c4d-f905-4a3b-8f8e-bd5d706d070d.png)

Have you tried updating webstorm? Are you using the 4.3.5 version of TypeScript in the IDE? You should trust only eslint or TS errors, not webstorm "warnings"

Reply:
TS is fine, I'll update the IDE for the 100th time thanks.

![image](https://user-images.githubusercontent.com/3502075/129848853-d1a66a33-a509-4527-a754-f07be7ed6493.png)